core/local/steps/initial_diff.js

```diff
@@ -137,18 +139,26 @@ async function initialDiff (buffer /*: Buffer */, out /*: Buffer */, pouch /*: P
               path: was.path
             })
           }
-        } else if (foundUntouchedFile(event, was)) {
+        } else if (hasMd5sum(was) && foundUntouchedFile(event, was)) {
```

Comment:
Maybe find a way to include all the conditional logic in `foundUntouchedFile()`?